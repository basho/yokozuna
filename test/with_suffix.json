{"name":"ryan",
 "age":29,
 "pets":["smokey", "bandit"],
 "books":[
     {"title":"Introduction to Information Retrieval",
      "authors":["Christopher D. Manning",
                 "Prabhakar Raghavan",
                 "Hinrich Schütze"]},
     {"title":"Principles of Distributed Database Systems",
      "authors":["M. Tamer Özsu", "Patrick Valduriez"]}
 ],
 "type":null,
 "alive":true,
 "married":false,
 "a_number":1.1e6,
 "lucky_numbers":[13,17,21],
 "misc":{},
 "kids":[]
}
