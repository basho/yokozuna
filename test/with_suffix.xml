<?xml version="1.0"?>
<add>
<doc>
<field name="lucky_numbers_ds">21</field>
<field name="lucky_numbers_ds">17</field>
<field name="lucky_numbers_ds">13</field>
<field name="a_number_d">1.10000000000000000000e+06</field>
<field name="married_b">false</field>
<field name="alive_b">true</field>
<field name="books_authors_txt">Patrick Valduriez</field>
<field name="books_authors_txt">M. Tamer Özsu</field>
<field name="books_title_txt">Principles of Distributed Database Systems</field>
<field name="books_authors_txt">Hinrich Schütze</field>
<field name="books_authors_txt">Prabhakar Raghavan</field>
<field name="books_authors_txt">Christopher D. Manning</field>
<field name="books_title_txt">Introduction to Information Retrieval</field>
<field name="pets_txt">bandit</field>
<field name="pets_txt">smokey</field>
<field name="age_d">29</field>
<field name="name_t">ryan</field>
</doc>
</add>
