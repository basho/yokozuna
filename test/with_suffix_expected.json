{"add":
{"doc":{
"lucky_numbers_ds":"21",
"lucky_numbers_ds":"17",
"lucky_numbers_ds":"13",
"a_number_d":"1.10000000000000000000e+06",
"married_b":"false",
"alive_b":"true",
"books_authors_txt":"Patrick Valduriez",
"books_authors_txt":"M. Tamer \u00d6zsu",
"books_title_txt":"Principles of Distributed Database Systems",
"books_authors_txt":"Hinrich Sch\u00fctze",
"books_authors_txt":"Prabhakar Raghavan",
"books_authors_txt":"Christopher D. Manning",
"books_title_txt":"Introduction to Information Retrieval",
"pets_txt":"bandit",
"pets_txt":"smokey",
"age_d":"29",
"name_t":"ryan"
}}}